package.json

```diff
@@ -137,13 +138,14 @@
     "tap": "15.0.9",
     "tape": "5.3.1",
     "tape-promise": "4.0.0",
-    "ts-jest": "28.0.2",
+    "ts-jest": "27.0.3",
```

Comment:
@micoferdinand98 Is this a rebase gone wrong? You seem to be downgrading a lot of unrelated dependencies and breaking the build.

Reply:
This was updated automatically when I pull the latest from the main branch, I'll  return it to original version now

Reply:
@micoferdinand98 Understood, thank you for the rebase. Please make sure to address the other review items as well!